### PR TITLE
Hey, I did my program i.e., reverse a string using stack in cpp

### LINKED ISSUE #98: Reversing a string using stacks
for example-prateek
output-keetarp

### PR DESCRIPTION
**Pull request by rahulsrivastava1**
Kindly check it out and do the needful.
Thankss!

Fixed #98

### CODE REVIEW

**Comment by tarun26091999** [comment]
link the issue.

**Comment by rahulsrivastava1** [comment]
How to link the issue? can you tell me please?

**Comment by rahulsrivastava1** [comment]
Fixes #98

File: Code/C++/reverse_a_string_using_stack.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	stack<int> str;
+	for(char ch:s)
+	{
+        str.push(ch);
+    }
+    for(i=0;i<s.size();i++)
+    {
+        s[i]=str.top();
+        str.pop();
+    }
```
Comment (tarun26091999) [inline Code/C++/reverse_a_string_using_stack.cpp:29]: do correct indentation.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
link it in the stack folder.

**Comment by rahulsrivastava1** [comment]
Hey @tarun26091999 sir, i fixed your concerns about this issue. kindly review it again.
Thanking You!

File: Code/C++/reverse_a_string_using_stack.cpp (hunk 1 above)
Comment (rahulsrivastava1) [reply Code/C++/reverse_a_string_using_stack.cpp]: Done

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
you haven't linked your code to the strings folder, also add comments that explain the steps.

**Review by tarun26091999 — APPROVED** [review]

**Review by raksha009 — APPROVED** [review]

**Pull request merged**